flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/table/RowDataKinesisDeserializationSchema.java

```diff
@@ -27,7 +27,7 @@
 import org.apache.flink.table.data.RowData;
 import org.apache.flink.table.data.StringData;
 import org.apache.flink.table.data.TimestampData;
-import org.apache.flink.table.data.utils.JoinedRowData;
+import org.apache.flink.table.data.JoinedRowData;
```

Comment:
Personally, I don't suggest to move package. 
1) I believe there are some external connectors are aready using it and this will break them. 
2) It was moved from `org.apache.flink.table.data` to `org.apache.flink.table.data.utils` in https://github.com/apache/flink/pull/13770#discussion_r518802767.

Reply:
Thanks Jark, that's a good point. I'll just remove the second commit.